restcomm/restcomm.application/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xmlns:web="http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd" id="WebApp_ID" version="2.5">
+<web-app version="2.5"
+         xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd">
   <display-name>RestComm</display-name>
-   
+
```

Comment:
@leftyb Restcomm web.xml doesn't need a "security-constraint" also? Similar to the web.xml of admin.ui and rvd.
I think that when we have HTTP redirected to HTTPS enabled, we want the requests to "restcomm.war" application to be redirected to HTTPS. For example REST API requestes, requests from RMS to get the audio files etc.

George